addon/components/phone-input.js

```diff
@@ -257,11 +257,24 @@ export default Component.extend({
     const extension = iti.getExtension();
     const selectedCountryData = iti.getSelectedCountryData();
     const isValidNumber = iti.isValidNumber();
+    // From https://github.com/jackocnr/intl-tel-input/blob/master/src/js/utils.js#L102
+    const E164 = iti.getNumber(0);
+    const INTERNATIONAL = iti.getNumber(1);
+    const NATIONAL = iti.getNumber(2);
+    const RFC3966 = iti.getNumber(3);
```

Comment:
Changes look overall good to me @ctjhoa. Thank you for this!

I would prefer to use `intlTelInputUtils.numberFormat` rather than directly an integer (might require to disable linting for intlTelInputUtils global). What do you think?

```suggestion
    const E164 = iti.getNumber(intlTelInputUtils.numberFormat.E164);
    const INTERNATIONAL = iti.getNumber(intlTelInputUtils.numberFormat.INTERNATIONAL);
    const NATIONAL = iti.getNumber(intlTelInputUtils.numberFormat.NATIONAL);
    const RFC3966 = iti.getNumber(intlTelInputUtils.numberFormat.RFC3966);
```

Reply:
I'm ok with that, I used integers as `intl-tel-input` already [copy paste](https://github.com/jackocnr/intl-tel-input/blob/master/src/js/utils.js#L102) them from [libphonenumber](https://github.com/google/libphonenumber/blob/master/javascript/i18n/phonenumbers/phonenumberutil.js#L900).

Reply:
done in fixup